projects/lights_control/inc/lights_events.h

```diff
@@ -19,21 +19,22 @@ typedef enum {
   LIGHTS_EVENT_STROBE_ON,
   // Event processed by lights_sync module.
   LIGHTS_EVENT_SYNC,
+  LIGHTS_EVENT_BPS,
```

Comment:
Should probably use `bps fault` instead of `bps`.

Reply:
`bps_heartbeat` sounds better because I'm planning to pass this to a `lights_watchdog` module.